src/lib/project.js

```diff
@@ -239,6 +239,7 @@ async function step(step, cmd) {
     spin.succeed(green(step));
   } catch (err) {
     spin.fail(step);
+    process.exit();
```

Comment:
If we omit the error code in `process.exit()`, I believe it returns `0` which is expected to be successful. We should instead do `process.exit(1)` to indicate a failure here in the returned error code.

Reply:
+1 I just changed it